pkg/types/ovirt/machinepool.go

```diff
@@ -24,6 +24,11 @@ type MachinePool struct {
 	// +kubebuilder:validation:Enum="";desktop;server;high_performance
 	// +optional
 	VMType VMType `json:"vmType,omitempty"`
+
+	// AffinityGroupsNames contains the name of the OpenShift cluster affinity groups
```

Comment:
Suggestion:

> AffinityGroupNames contains a list of oVirt affinity group names that will be used to separate the OpenShift cluster nodes from each other to separate physical machines. If the affinity group is not created in the install config it is assumed that the AG already exists in oVirt.

Reply:
Changed it to:
```
AffinityGroupsNames contains a list of oVirt affinity group names that the newly created machines will join.
The affinity groups should exist on the oVirt cluster or created by the OpenShift installer.
```
I don't want to say for what the affinity group will be used for :)